src/main/resources/application.properties

```diff
@@ -1 +1,12 @@
 server.port=9090
+
+# PostgreSQL
+spring.datasource.url=jdbc:postgresql://${DATABASE_ENDPOINT}:${DATABASE_PORT}/${DATABASE_NAME}?createDatabaseIfNotExist=true&autoReconnect=true&useSSL=false
```

Comment:
```
createDatabaseIfNotExist=true&autoReconnect=true&useSSL=false
```
Not a blocker but out of curiosity do you have a view on whether these query parameters will apply to all of our environments? If potentially not I’m wondering if we aren’t better storing the whole database connection string in the variable, rather than having the env vars for the individual components and building it here. I don’t _think_ we get any special value from having them separate 🤔

Reply:
I'm not 100% certain if they will be different across the other environments. We took those properties from the callisto k8's exploration liquibase example which doesn't mention if it's environment specific.

I think it should be ok to just put the entire connection string as an env var as you've suggested anyway though.

Reply:
We agreed on a call that application.properties was the best place for this after all. Building a connection string that makes decisions about how the database is accessed and keeping it in helm or even worse k8s secrets feels too far away from the app code to be sensible